### PR TITLE
fix(docs): updated changelog and README.md version 3.2.0

### PR DESCRIPTION
**Pull request by SvetlanaMuravlova**
fix(docs): updated changelog and README.md version 3.2.0

### CODE REVIEW

**Comment by ignaciostellino** [comment]
@SvetlanaMuravlova I am trying to use version 3.1.0 and using angular 13 and I am getting error could not find Cannot find module 'ng2-dragula' or its corresponding type declarations when importing DragulaService. Any ideas?

**Comment by SvetlanaMuravlova** [comment]
> @SvetlanaMuravlova I am trying to use version 3.1.0 and using angular 13 and I am getting error could not find Cannot find module 'ng2-dragula' or its corresponding type declarations when importing DragulaService. Any ideas?

yah, it could be, because of it, use, please version 3.2.0

File: README.md [hunk 1]
```diff
@@ -43,10 +44,10 @@ Or play with [this starter in your browser][stackblitz] on StackBlitz.
...
 Latest version available for each version of Angular
 
-| ng2-dragula | Angular    |
-| ---------- | ----------- |
-| 2.1.1      | <= 9.x      |
-| current    | >= 10.x     |
+| ng2-dragula | Angular |
+| ---------- |---------|
+| 2.1.1      | <= 9.x  |
+| current    | ^14.x.x |
```
Comment (rg1220) [inline README.md:50]: It seems that it is being built this way, but is there a reason it's restricted to `^14.x.x`? I'm using a project that just upgraded to Angular 15 and now `ng2-dragula` is failing.

Comment (SvetlanaMuravlova) [reply README.md:50]: yeah, our library hasn't been updated yet, so now it fails. It will be updated soon

**Pull request closed without merging**